metrics/test/features/dashboard/domain/usecases/receive_build_metrics_updates_test.dart

```diff
@@ -8,7 +10,7 @@ import 'package:metrics/util/date.dart';
 import 'package:test/test.dart';
 
 void main() {
-  group("Build metrics data loading", () {
+  group("ReceiveProjectMetricUpdates", () {
```

Comment:
Can we add more edge cases? Like all failed builds loaded and we need to find build for coverage?